### PR TITLE
Add Hyper Orama plugin

### PR DESCRIPTION
**Pull request by cjoshmartin**
This plugin adds support for screen recording directly from hyper and uploading to Now. 

w/  @juancampa , @tejasq , @kmui2 , @cwlowder , @coolcodehere,

### CODE REVIEW

**Comment by juancampa** [comment]
Do not merge this yet. It's not on NPM yet.

**Comment by MAPESO** [comment]
@cjoshmartin Hi 👋, a quick question. What format does the recording carry ?

**Comment by MAPESO** [comment]
@cjoshmartin Hi, a quick question. What format is the recording exported ? 🤔

**Comment by TejasQ** [comment]
Hey @MAPESO, it's `webp`. 😄

**Comment by cjoshmartin** [comment]
So, the package is on npm now but is under the name  [hyper--orama](https://www.npmjs.com/package/hyper--orama/v/0.0.1) and has 68 downloads. 

1) Should I change the name of the npm package to `hyper-orama` ?
2) Is this PR good to merge?

**Comment by juancampa** [comment]
@cjoshmartin can you publish it to `hyper-orama` now that it shouldn't be blocked? Once that's published we can merge this

**Comment by cjoshmartin** [comment]
@juancampa I fixed it!

**Comment by MAPESO** [comment]
cc @timothyis we can make merge of this 😸

**Comment by rauchg** [comment]
Can we make the toolbar look more accurate with regards to Hyper's style?

Recording:

image

Expected:

image

**Comment by MAPESO** [comment]
@rauchg  Good observation 🤔

**Comment by juancampa** [comment]
cc @rioam2 :point_up_2:

**Comment by rioam2** [comment]
@rauchg I can make some modifications to the frame when I have a spare moment :)

**Comment by cjoshmartin** [comment]
@rioam2 Did fix this in https://github.com/cwlowder/hyper-orama/pull/62 ?

**Comment by rioam2** [comment]
> @rioam2 Did fix this in [cwlowder/hyper-orama#62](https://github.com/cwlowder/hyper-orama/pull/62) ?

Yes, the PR addressed this and the plugin's branding now matches up with what's expected 😄

File: plugins.json [hunk 1]
```diff
@@ -583,5 +583,13 @@
     ],
     "preview": "https://github.com/manniefesto/hyper-cyberpunk/raw/2ae9b73949441f2526279b1acd213377aceb4d7e/cyberpunk.gif?raw=true",
     "dateAdded": "1547219532507"
+  },
+	
+  {
+    "name": "hyper-orama",
+    "description": "Record and share your Hyper terminal sesson",
+   "type": "plugin",
+    "preview": "https://github.com/cwlowder/hyper-orama/blob/master/demo-out.gif?raw=true",
```
Comment (rioam2) [inline plugins.json]:
```suggestion
    "preview": "https://user-images.githubusercontent.com/35539750/54399585-10efe100-468d-11e9-9822-e0220a8e3152.gif"
```

Comment (cjoshmartin) [reply plugins.json]: CC @rauchg @juancampa @MAPESO @TejasQ

Comment (TejasQ) [reply plugins.json]: I agree.

**Comment by TejasQ** [comment]
Can we either merge or close this please?

**Comment by timothyis** [comment]
Thanks for the ping @TejasQ. @vassbence, would you mind testing this one?

**Comment by TejasQ** [comment]
Please close this if it isn't going to get the attention it needs.

**Comment by cwlowder** [comment]
@TejasQ hello again! I might take a crack at finishing this up if @cjoshmartin is up for it?

**Comment by cjoshmartin** [comment]
@cwlowder I am up to it, Do I need to push to the npm package?

**Comment by vassbence** [comment]
image
Could not test it out, I get this error message on every startup since I installed the plugin with `hyper i hyper-orama`. Let me know if you need some data from me to debug this.

**Comment by cjoshmartin** [comment]
@vassbence I will look into it more, but from a quick search it seems to a problem with the store: https://github.com/cwlowder/hyper-orama/search?q=window&unscoped_q=window

**Comment by cwlowder** [comment]
@cjoshmartin I'm going to set aside time to do this all this weekend

**Comment by TejasQ** [comment]
Can we please close this?

**Comment by TejasQ** [comment]
Can we please close this?

**Comment by cjoshmartin** [comment]
@TejasQ yeah, I guess just close it

**Pull request closed without merging**